### PR TITLE
Auto select single location

### PR DESCRIPTION
**Pull request by jsmylnycky**

### CODE REVIEW

**Comment by darushka** [comment]
:+1:

File: dist/peach.js [hunk 1]
```diff
@@ -3274,7 +3274,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	      if (this.$peach.session.getActiveAccount()) {
 	        this.activateAllApps();
 	        this.activateShowDashboard();
-	        this.activateShowWelcome();
+	        //this.activateShowWelcome();
```
Comment (againsley) [inline dist/peach.js]: Why not just remove these?

**Comment by rynomars** [comment]
:thumbsup:

**Comment by againsley** [comment]
:+1:

File: dist/peach.js (hunk 1 above)
Comment (jsmylnycky) [inline dist/peach.js]: @againsley It'll be used once we get the welcome app developed

**Comment by jsmylnycky** [comment]
Released v0.4.2

**Pull request closed without merging**